src/client/content_cao.h

```diff
@@ -263,4 +264,12 @@ class GenericCAO : public ClientActiveObject
 	{
 		return m_prop.infotext;
 	}
+
+	bool doShowSelectionBox() override
+	{
+		if (m_client && m_client->getGameUI())
```

Comment:
```C++
return m_prop.show_selection_box || (m_client && m_client->getGameUI()->getFlags().show_debug);
```
1) Slightly faster due to short-circuit `OR` if the first condition is `true`
2) Removes superfluous `m_client->getGameUI()` check because this cannot be `nullptr` (see `game.cpp`)